### PR TITLE
progressbar

### PR DESCRIPTION
**Pull request by Amitrawat21**
progress bar

### CODE REVIEW

File: src/components/progressBar/ProgressBar.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { View, Text} from 'react-native';
+import { ProgressBarComponentProps } from '../../interface/ProgressbarInterface/ProgressBar';
+import { responsiveHeight, responsiveWidth } from 'react-native-responsive-dimensions';
+
+import Slider from '@react-native-community/slider';
+
```
Comment (tabassum2507) [inline src/components/progressBar/ProgressBar.tsx:7]: remove extra spaces

File: src/components/progressBar/ProgressBar.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+export const ProgressBarComponentComponent:React.FC<ProgressBarComponentProps> = ({
```
Comment (tabassum2507) [inline src/components/progressBar/ProgressBar.tsx]: change props to interface

Comment (tabassum2507) [inline src/components/progressBar/ProgressBar.tsx]: ProgressBarComponentComponent? remove one component

File: src/components/rangepicker/index.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
 export { default as ContainedButton } from '../buttons/containedButton';
-
+import { ProgressBarComponentProps } from '../../interface/ProgressbarInterface/ProgressBar';
 // export all types of buttons here
```
Comment (tabassum2507) [inline src/components/rangepicker/index.ts]: why RangePicker?

File: src/interface/ProgressbarInterface/ProgressBar.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+  export interface ProgressBarComponentProps {
```
Comment (tabassum2507) [inline src/interface/ProgressbarInterface/ProgressBar.ts]: make all the props mandatory

**Review by tabassum2507 — CHANGES_REQUESTED** [review]

File: src/components/progressBar/ProgressBar.tsx (hunk 2 above)
Comment (rohan-arya-w) [reply src/components/progressBar/ProgressBar.tsx]: Still props

File: src/components/progressBar/ProgressBar.tsx (hunk 1 above)
Comment (Amitrawat21) [reply src/components/progressBar/ProgressBar.tsx:7]: resolved

File: src/interface/ProgressbarInterface/ProgressBar.ts (hunk 4 above)
Comment (Amitrawat21) [reply src/interface/ProgressbarInterface/ProgressBar.ts]: resolved

**Review by rohan-arya-w — APPROVED** [review]

File: src/components/rangepicker/index.ts (hunk 3 above)
Comment (Amitrawat21) [reply src/components/rangepicker/index.ts]: resolved

File: src/components/progressBar/ProgressBar.tsx (hunk 2 above)
Comment (Amitrawat21) [reply src/components/progressBar/ProgressBar.tsx]: resolved

**Review by tabassum2507 — APPROVED** [review]

**Pull request merged**